RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -26,13 +26,17 @@ import SwiftUI
 @available(visionOS, unavailable)
 public struct CustomerCenterView: View {
 
-    @StateObject private var viewModel = CustomerCenterViewModel()
+    @StateObject
+    private var viewModel = CustomerCenterViewModel()
+    @StateObject
+    private var completionHandler: CustomerCenterCompletionHandler = .default()
```

Comment:
Would it be possible to move this inside the view model? That way we centralize the logic there.